lib/ManualImageCropEditorWindow.php

```diff
@@ -94,6 +94,15 @@ public function renderWindow() {
 				$src_file = str_replace($uploadsDir['baseurl'], $uploadsDir['basedir'], $src_file_url[0]);
 				$sizes = getimagesize($src_file);
 
+				$original[0] = $sizes[0];
+				$original[1] = $sizes[1];
+
+				if ($width > $sizes[0]) {
```

Comment:
$original is just for echo in _e('Original picture dimensions:','microp') on "summary".

if ($width > $sizes[0]) { opens scaling to wider image only if needed.